data/resources.json

```diff
@@ -45,7 +45,12 @@
           "newTab": false
         },
         {
-          "name": "College for Creative Studies",
+          "name": "College for Creative Studies,",
+          "url": "https://www.collegeforcreativestudies.edu",
+          "newTab": true
+        },
+        {
+          "name": "A. Alfred Taubman Center for Design Education.",
```

Comment:
do we want to link to both of these places, or just one of them?

Reply:
I've put in the same link twice because I couldn't find another way of splitting the full title "College for Creative Studies, A. Alfred Taubman Center for Design Education." neatly over two lines. (`\n` and `<br>` get auto escaped on injection.) I guess we could directly hardcode the footer links instead of injecting from resources.json... feels like a step backward though.

Reply:
I agree, and it's weird because I've used HTML formatting elsewhere in resources.json but verified that in this case it's getting escaped. Couldn't figure out why, but not really a big deal.